puppet/ha-puppet/screenshot.js

```diff
@@ -144,30 +151,8 @@ export class Browser {
       // Open a lightweight page to set local storage
       await page.goto(`${hassUrl}/robots.txt`);
 
-      // Store access token in local storage when page is opened
-      await page.evaluate(
-        (hassUrl, clientId, token, hassLocalStorage) => {
-          for (const [key, value] of Object.entries(hassLocalStorage)) {
-            localStorage.setItem(key, value);
-          }
-          localStorage.setItem(
-            "hassTokens",
-            JSON.stringify({
-              access_token: token,
-              token_type: "Bearer",
-              expires_in: 1800,
-              hassUrl,
-              clientId,
-              expires: 9999999999999,
-              refresh_token: "",
-            }),
-          );
-        },
-        hassUrl,
-        clientId,
-        this.token,
-        hassLocalStorageDefaults,
-      );
+      await this.initializeLocalStorage(page, hassUrl, clientId, darkMode);
```

Comment:
Please keep this method inlined. No need to extract methods unnecessarily. Makes code harder to follow.